components/app/interface/settings/client/index.tsx

```diff
@@ -196,6 +197,14 @@ const Config: SettingsConfiguration<{ server: Server }> = {
           {
             id: "logout",
             icon: <BiSolidExit size={20} color={theme!.colours.error} />,
+            href: "/login",
+            onClick: () => {
+              const acli = getController("state");
+              const cli = getController("client").getCurrentClient();
+              console.log(cli);
+              acli.auth.removeSession(cli?.user?.id as string);
```

Comment:
This will remove the session from storage but not kill the active client in the client controller, which is remedied by assigning `window.location.href` (since that will just kill all active sessions) but we want to avoid reloading the client.